server/index.js

```diff
@@ -23,7 +23,6 @@ import pkg from '../../package'
 export default class Server {
   constructor ({ dir = '.', dev = false, staticMarkup = false, quiet = false, conf = null } = {}) {
     this.dir = resolve(dir)
-    this.dev = dev
     this.quiet = quiet
     this.router = new Router()
     const phase = dev ? PHASE_DEVELOPMENT_SERVER : PHASE_PRODUCTION_SERVER
```

Comment:
you could probably get rid of `dev` if you use the constructor only to define instance configuration and then add a `init` method to the class which you'd call explicitly. This way the dev server could do the extra work on its constructor

Reply:
Yeah, this is just the first wave of changes, I'm going to progressively move more code to the dev server 👍

Reply:
Fixed 🙌